packages/react-next/src/components/Pivot/Pivot.base.tsx

```diff
@@ -138,7 +139,7 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef(
           {...headerButtonProps}
           id={tabId}
           key={itemKey}
-          className={isSelected ? classNames.linkIsSelected : classNames.link}
+          className={css(classNames.link, isSelected && classNames.linkIsSelected)}
           onClick={onLinkClick.bind(this, itemKey)}
           onKeyPress={onKeyPress.bind(this, itemKey)}
```

Comment:
is it intentional to use onKeyPress instead of onKeyDown which is typically used for keyboard handling?

Reply:
since this is not convergence yet, feel free to ignore. but it might still be worth checking.

Reply:
This is out of scope for this PR, but I logged #13429 to fix this in the future. Thanks.